truss/model_inference.py

```diff
@@ -97,7 +97,7 @@ def validate_provided_parameters_with_model(
         model_class: The model class to validate against
         provided_parameters: The parameters to validate
     """
-    if type(model_class) is ClassDef:
+    if type(model_class) == ClassDef:
```

Comment:
What about `issubclass`, does it not work here?

Reply:
To me it looks like this method is not even used, so I reverted it to how it was before

Reply:
Good call, let's always remove dead code.